nxcompshad/src/Core.cpp

```diff
@@ -229,7 +229,7 @@ int CorePoller::isChanged(int (*checkIfInputCallback)(void *), void *arg, int *s
   // 
 
   static int idxIlace = 0;
-  static int curLine = 0;
+  static short int curLine = 0;
```

Comment:
The best way to handle this situation would probably be to keep this an `int` and check the value that it doesn't overflow or underflow `unsigned short` when passing it to other stuff that expects an `unsigned short`. But would also be a lot more invasive.